docs/usage.md

```diff
@@ -431,14 +431,27 @@ We can also write these references out as an [IcechunkStore](https://icechunk.io
 ```python
 # create an icechunk repository, session and write the virtual dataset to the session
 from icechunk import Repository, Storage, VirtualChunkContainer, local_filesystem_storage
-storage = local_filesystem_storage(str('combined'))
```

Comment:
I wasn't really sure why the argument was `str('combined')` but happy to revert this change if there is a reason

Reply:
I mean this can't possibly be necessary, because `str(<str obj>) == <str obj>`, even down to having the same hash. So you should be able to get rid of the enclosing `str()` call.

Reply:
oh yes, I did already get rid of the enclosing `str` call (see change below), but I also changed the string, which I think more clearly indicates a local path